calculator.js

```diff
@@ -115,24 +116,22 @@ function handlesNumber(value){
         btnDecSeparator.classList.remove('dot-disabled');
         clearScreen();
     }        
-    showOnScreen(value);
+    showOnScreen(value,'add');
 }
 
 function handlesOperator(value){
 
     if (!operator){
         num1 = digitsOnScreen.innerHTML;
-    }else if(num1) {
+    }else if(num1 && !operatorState) {
```

Comment:
`operatorState` is means that a signal was clicked right?
In the scenario:
- Click number
- Click plus 
- Click number

Will this continue to work?

Reply:
Happy path still works, QA tested